src/hooks/v2/transactor/LaunchProjectWithNftsTx.ts

```diff
@@ -70,7 +72,7 @@ async function getJBDeployTiered721DelegateData({
     symbol: collectionSymbol,
     tokenUriResolver: constants.AddressZero,
     baseUri: ipfsCidUrl(''),
-    contractUri: 'ipfs://null',
+    contractUri: ipfsCidUrl(collectionCID),
```

Comment:
Link to collection's metadata on IPFS. e.g. https://jbx.mypinata.cloud/ipfs/QmXKh4DECXsBWx3FRX8tM3LUkxewnZrnhLLaBNPASzq657